setup.py

```diff
@@ -20,34 +18,25 @@
     "elasticsearch-dsl==5.4.0",
     "elasticsearch==5.4.0",
     "future==0.18.2",
-    "humanfriendly==6.1",
-    "jsonschema-serialize-fork @ git+https://github.com/lrowe/jsonschema_serialize_fork.git@2.1.1",  # noqa
+    "jsonschema_serialize_fork==2.1.1",
```

Comment:
underscores not dashes?

Reply:
yes: https://pypi.org/project/jsonschema_serialize_fork/